docs/ContributionGuide.md

```diff
@@ -20,7 +20,7 @@
 ## Prerequisites
 
 Minimum requirements are:
- - **Node.js** version 12.
+ - **Node.js** version 14.
```

Comment:
Is 14 really our minimum? I find it always better if it says for example 12 and above for example.

Reply:
Yes. we have written Optional Chaining in the code, which is supported in node 14 or above.